Define the end of the competition (#1050)
thewca/wca-regulations: wca-guidelines.md

- 9t+) [EXPLANATION] Because of [9t1](regulations:regulation:9t1) and [9t2](regulations:regulation:9t2), rounds may look like they do not comply with [9p1](regulations:regulation:9p1), [9m1](regulations:regulation:9m1), [9m2](regulations:regulation:9m2) or [9m3](regulations:regulation:9m3). However, making changes to comply to these rules would affect innocent competitors, which is why the results must stay the same.
- 9t++) [ADDITION] If the change of results happens during the competition, this rule must also be applied. In that case, [9t1](regulations:regulation:9t1), [9t2](regulations:regulation:9t2) and [9t3](regulations:regulation:9t3) should also be followed, at the discretion of the WCA Delegate.
- 9t3+) [EXAMPLE] Suppose competitor A gets a 10.00 average and competitor B gets a 10.05 average in the first round of an event, such that competitor A advances to the final round but not B. If a score-taking mistake is discovered, and competitor B actually got a 9.95 average (and should have proceeded instead of A), competitor A keeps their result for the final round.

> - 9u+) [ADDITION] In cases such as a FMC multi-location competition, all the requirements must be met in every location to consider the competition as ended.

Reviewer: Nit: we don't use the abbreviation "FMC" in the Regulations. https://www.worldcubeassociation.org/documents/policies/external/Competition%20Requirements.pdf uses "3x3x3 Fewest Moves simultaneous competitions".

```suggestion
- 9u+) [ADDITION] For competition with multiple physical locations (e.g. "3x3x3 Fewest Moves simultaneous competitions"), all the requirements must be met in every location to consider the competition as ended.
```

> - 9u+) [ADDITION] In cases such as a FMC multi-location competition, all the requirements must be met in every single location to consider the competition as ended.

Reviewer 2: ```suggestion
- 9u+) [ADDITION] In cases such as a FMC multi-location competition, all the requirements must be met in every location to consider the competition as ended.
```

- 9u2+) [RECOMMENDATION] The WCA Delegate should announce the end of the competition (e.g. prior to the awards ceremony).

> - 9u2b+) [ADDITION] The local time for the location in which the competition took place must be considered to determine the end of the calendar date.

Reviewer: In case of multiple locations across time zones?, presumably this this should be the latest of any local times?
```suggestion
- 9u2b+) [ADDITION] The calendar date of the end of the competition is determined by the local time for the location in which the competition took place.
```
